buildman/manager/ephemeral.py

```diff
@@ -566,6 +566,12 @@ def schedule(self, build_job):
                 realm,
                 build_uuid,
             )
+        except OrchestratorConnectionError:
+            logger.exception(
```

Comment:
Does this not need to return or re-raise?

Reply:
The above one was not being re-raised or returned, so I'm just logging them, since these are just for job metrics.